src/hotspot/share/memory/virtualspace.cpp

```diff
@@ -702,6 +705,7 @@ bool VirtualSpace::initialize_with_granularity(ReservedSpace rs, size_t committe
   assert(_low_boundary == nullptr, "VirtualSpace already initialized");
   assert(max_commit_granularity > 0, "Granularity must be non-zero.");
 
+  _nmt_flag = rs.nmt_flag();
```

Comment:
The code seems to be written with blank lines to separate various members that belong together. Please add a blank line after this line.

Reply:
Fixed and moved.